packages/driver/cypress/fixtures/xhr-fetch-onload.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <body>
+    <h1 data-cy="assertion-header">Making XHR and Fetch Requests behind the scenes!</h1>
+    <script>
+      function fireXHRAndFetchRequests() {
+
+        xhr = new XMLHttpRequest();
+        xhr.open("GET", "https://jsonplaceholder.cypress.io/todos/1");
```

Comment:
can we make a fetch request to our local server instead of to .cypress.io? I see us using this in a couple of other places already, but the seems simple enough that we could add it to the server.

Reply:
Made a fixture and stubbed the body out with `cy.intercept` in 265617f3cc. Seems to do what we want?